### PR TITLE
fix: remove user stories

### PR DESCRIPTION
**Pull request by moT01**
Related to: https://github.com/freeCodeCamp/freeCodeCamp/issues/39721

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,2 @@
...
-1. I can get the IP address, preferred languages (from header `Accept-Language`) and system infos (from header `User-Agent`) for my device.
-
-#### Example usage:
-* [base_url]/api/whoami
-
-#### Example output:
-* `{"ipaddress":"159.20.14.100","language":"en-US,en;q=0.5","software":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:50.0) Gecko/20100101 Firefox/50.0"}`
\ No newline at end of file
+# [Request Header Parser Microservice](https://www.freecodecamp.org/learn/apis-and-microservices/apis-and-microservices-projects/request-header-parser-microservice
+)
```
Comment (Sky020) [inline README.md:2]:
```suggestion
# [Request Header Parser Microservice](https://www.freecodecamp.org/learn/apis-and-microservices/apis-and-microservices-projects/request-header-parser-microservice)
```

**Review by Sky020 — APPROVED** [review]
LGTM 👍 Pending decision on whether to remove hyperdev stuffs or not

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**